lib/danger/danger_suggester.rb

```diff
@@ -1,5 +1,5 @@
 require 'danger'
-require 'danger/suggester/file'
+require "danger/suggester/file"
```

Comment:
```suggestion
require 'danger/suggester/file'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>